src/qt/askpassphrasedialog.cpp

```diff
@@ -234,6 +237,15 @@ bool AskPassphraseDialog::event(QEvent *event)
     return QWidget::event(event);
 }
 
+void AskPassphraseDialog::toggleShowPassword(bool show)
+{
+    ui->toggleShowPasswordButton->setDown(show);
```

Comment:
I haven't tested, but since the button is `checkable` this shouldn't be necessary right?

Reply:
It did not visually switch to being 'down' when toggled until I added the setDown() call.  It seems to require both the checkable = true and the call to setDown().